assets/pages/monthly-contributions/components/formFields.jsx

```diff
@@ -6,23 +6,53 @@ import React from 'react';
 import { connect } from 'react-redux';
 
 import TextInput from 'components/textInput/textInput';
+import SelectInput from 'components/selectInput/selectInput';
 
 import {
   setFirstName,
   setLastName,
+  setStateField,
 } from 'helpers/user/userActions';
+import { usStates } from 'helpers/internationalisation/country';
+
+import type { IsoCountry, UsState } from 'helpers/internationalisation/country';
+import type { SelectOption } from 'components/selectInput/selectInput';
 
 
 // ----- Types ----- //
 
 type PropTypes = {
   firstNameUpdate: (name: string) => void,
   lastNameUpdate: (name: string) => void,
+  stateUpdate: (value: UsState) => void,
   firstName: string,
   lastName: string,
+  country: IsoCountry,
 };
 
 
+// ----- Functions ----- //
+
+function stateDropdown(country: IsoCountry, stateUpdate: UsState => void) {
+
+  if (country === 'US') {
+
+    const options: SelectOption[] = Object.keys(usStates).map((stateCode: UsState) =>
+      ({ value: stateCode, text: usStates[stateCode] }),
+    );
+
+    // Sets the initial state to the first in the dropdown.
+    stateUpdate(options[0].value);
```

Comment:
Discussed offline. We decided to make Alaska the default to make it clearer that this is a State dropdown that requires input. There is a certain amount of complexity involved in validating an empty State field due to the lack of control we have over the PayPal button (rabbit-hole).